### PR TITLE
Feature/language clinician

### PR DESCRIPTION
**Pull request by quentingirard**

### CODE REVIEW

File: src/Services/Algorithm/FetchOne.js [hunk 1]
```diff
@@ -14,10 +14,10 @@ import axios from 'axios'
  * The internal imports
  */
 import api from '@/Services/Algorithm/FetchOneApi'
-import ChangeLanguage from '@/Store/System/ChangeLanguage'
+// import ChangeLanguage from '@/Store/HealthFacility/ChangeLanguage'
```
Comment (ManuBarchich) [inline src/Services/Algorithm/FetchOne.js:17]: :hankey:

File: src/Services/Algorithm/FetchOne.js [hunk 2]
```diff
@@ -14,10 +14,10 @@ import axios from 'axios'
...
 import { store } from '@/Store'
 import { Config } from '@/Config'
-import i18n from '@/Translations/index'
+// import i18n from '@/Translations/index'
```
Comment (ManuBarchich) [inline src/Services/Algorithm/FetchOne.js:21]: :hankey:

File: src/Services/Algorithm/FetchOne.js [hunk 3]
```diff
@@ -91,29 +91,29 @@ export default async ({ json_version }, { dispatch }) => {
...
-    ) {
-      await dispatch(
-        ChangeLanguage.action({
-          key: 'algorithmLanguage',
-          newLanguage: algorithm.study.default_language,
-        }),
-      )
-    }
-  }
+  // // TODO REMOVE IN NEXT RELEASE IF STATEMENT
```
Comment (ManuBarchich) [inline src/Services/Algorithm/FetchOne.js:94]: :hankey:

File: src/Services/HealthFacility/FetchOne.js [hunk 4]
```diff
@@ -36,5 +38,49 @@ export default async ({}) => {
...
+
+  // Update medical staff
+  if (currentHealthFacility.hasOwnProperty('medical_staffs')) {
+    medicalStaffs = response.data.medical_staffs.map(newMedicalStaff => {
+      const storedMedicalStaff = currentHealthFacility.medical_staffs.find(
+        ms => ms.id === newMedicalStaff.id,
+      )
+
+      // Existing staff but updated in medAL-c
+      if (
```
Comment (ManuBarchich) [inline src/Services/HealthFacility/FetchOne.js:56]: if (storedMedicalStaff)
  if (storedMedicalStaff.updated_at !== newMedicalStaff.updated_at)
  else
  end
else
end

File: src/Store/HealthFacility/index.js [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
 }
 
 export default buildSlice(
   'healthFacility',
-  [FetchOne, ChangeClinician, Destroy],
+  [FetchOne, ChangeClinician, Destroy, ChangeLanguage],
   sliceInitialState,
 ).reducer
+
+// appLanguage: 'en',
```
Comment (ManuBarchich) [inline src/Store/HealthFacility/index.js:21]: :hankey:

**Review by ManuBarchich — CHANGES_REQUESTED** [review]

File: src/Store/HealthFacility/index.js (hunk 5 above)
Comment (cini9) [inline src/Store/HealthFacility/index.js:21]: 💩

**Review by cini9 — CHANGES_REQUESTED** [review]

File: src/Services/HealthFacility/FetchOne.js (hunk 4 above)
Comment (quentingirard) [reply src/Services/HealthFacility/FetchOne.js:56]: no

**Review by cini9 — APPROVED** [review]

**Pull request closed without merging**